runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -223,15 +248,15 @@ private void calculate()
 			if (neededXP > 0)
 				actionCount = (int) Math.ceil(neededXP / xp);
 
-			slot.setText(formatXPActionString(xp, actionCount));
+			slot.setText("Lvl. " + slot.action.level + " (" + formatXPActionString(xp, actionCount));
 			slot.setAvailable(currentLevel >= slot.action.level);
 			slot.value = xp;
 		}
 	}
 
 	private String formatXPActionString(double xp, int actionCount)
 	{
-		return XP_FORMAT.format(xp) + "xp - " + NumberFormat.getIntegerInstance().format(actionCount) + (actionCount > 1 ? " actions" : " action");
+		return XP_FORMAT.format(xp) + "exp) - " + NumberFormat.getIntegerInstance().format(actionCount) + (actionCount > 1 ? " actions" : " action");
```

Comment:
remove the extra parenthesis

Reply:
Not extra. The string that comes before it has (